Makefile

```diff
@@ -17,3 +17,6 @@ catppuccinpalette-ctan.zip: catppuccinpalette.dtx
 
 install:
 	l3build install --full
+
+whiskers:
+	whiskers catppuccinpalette.dtx.tera > catppuccinpalette.dtx
```

Comment:
Instead of piping the output in this command, you can set `whiskers.filename` to `catppuccinpalette.dtx` in the template.